sqlstore/src/main/resources/db/migration/V8226__add_deliverables_to_project.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE Project ADD deliverableId bigint DEFAULT NULL;
```

Comment:
* this can go in the same migration as adding the Deliverable table
* add a foreign key constraint